src/components/PopUpBar/PopUpBar.css

```diff
@@ -1,5 +1,5 @@
 .popup_container {
-  position: flex;
+  position: fixed;
```

Comment:
Hi @LeeSingChanYau,  
The bar is not sticky it floats in middle.
Use 
```
position: sticky
top:0
```
it will solve the issue

Data is getting updated but it takes few seconds

check the video for more info


https://user-images.githubusercontent.com/82662768/216807562-2b493553-6201-46c6-aba8-a2e799301967.mp4

Reply:
@PrabhjotSembhi Yes, I believe this is what we want after confirming with @one-community. We want it to be there even when we scroll.

Reply:
Yes, even when we scroll. 

Thank you all for helping get this PR checked it, it's one of the longest outstanding ones.

Reply:
Hi @LeeSingChanYau,

My comment for PR #654 indicates the same issue, and thanks @PrabhjotSembhi for providing the solution!
In addition, there is a tiny issue: the lengths on both sides(red-circled in the screenshot below) of the bar are different, can you please make them symmetrical? Thanks!
![截屏2023-02-08 下午5 18 58](https://user-images.githubusercontent.com/77643813/217690910-38fda10f-98cd-4c57-8aad-353a5f7e6f14.png)

Reply:
@LeeSingChanYau what's The status on this?  I just want to know if are we implementing the changes and when we can complete this.
Thanks

Reply:
@LeeSingChanYau @NyahJ 

My solution will also fix that overflow
![pr595](https://user-images.githubusercontent.com/82662768/219414076-c27516b4-3d1a-4417-aafe-370d0630cb9d.jpg)